### PR TITLE
contrib/terminator: new package (2.1.3)

### PR DESCRIPTION
**Pull request by yopito**

### CODE REVIEW

File: contrib/libkeybinder3/template.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+source = (
+    f"https://github.com/engla/keybinder/archive/keybinder-3.0-v{pkgver}.tar.gz"
+)
+sha256 = "2eec50be6dfafa672d719e719f639df10403f270b2473e60deb2fb8455d13c51"
+hardening = ["vis"]
+# crossbuild fails: does not find gobject-introspection nor gtk-doc
+options = ["!cross"]
+
+
+def pre_configure(self):
```
Comment (q66) [inline contrib/libkeybinder3/template.py:29]: should be done with `configure_gen`

File: contrib/python-configobj/template.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+build_style = "python_module"
+hostmakedepends = ["python-setuptools"]
+depends = ["python-six"]
+pkgdesc = "Simple but powerful config file reader and writer"
+maintainer = "yopito <pierre.bourgin@free.fr>"
+license = "BSD-3-Clause"
+url = "https://github.com/DiffSK/configobj"
+source = f"https://github.com/DiffSK/configobj/archive/v{pkgver}.tar.gz"
+sha256 = "547dc047e31c71d7a8732016336769ed450588f34a7c13077aa7acc7df245eda"
+hardening = ["vis"]
```
Comment (q66) [inline contrib/python-configobj/template.py:13]: drop this, it's a python module without compiled parts

File: contrib/python-psutil/template.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+pkgname = "python-psutil"
+pkgver = "5.9.5"
+pkgrel = 0
+build_style = "python_module"
+hostmakedepends = ["python-setuptools", "python-devel"]
+makedepends = ["linux-headers"]
+depends = ["python"]
+pkgdesc = "Cross-platform process and system utilities module for Python3"
```
Comment (q66) [inline contrib/python-psutil/template.py:8]: should be something like "Process and system monitoring for Python"

File: contrib/terminator/template.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+]
+pkgdesc = "Tiling terminal emulator application"
+maintainer = "yopito <pierre.bourgin@free.fr>"
+license = "GPL-2.0-only"
+url = "https://gnome-terminator.org"
+source = f"https://github.com/gnome-terminator/terminator/releases/download/v{pkgver}/terminator-{pkgver}.tar.gz"
+sha256 = "0ae9943f3d6b72230c14bcb355de84dd81274c033e76ca4698e80d7c93cd6ae5"
+hardening = ["vis"]
+# crossbuild: wait for crossbuilable deps
+# checks needs gtk/gobject-instrospection and xvfb
```
Comment (q66) [inline contrib/terminator/template.py:34]: cannot be done with `xvfb-run`?

**Review by q66 — CHANGES_REQUESTED** [review]

File: contrib/libkeybinder3/template.py (hunk 1 above)
Comment (yopito) [reply contrib/libkeybinder3/template.py]: yes, thanks.

File: contrib/python-configobj/template.py (hunk 2 above)
Comment (yopito) [reply contrib/python-configobj/template.py]: my bad, thanks for pointing this

File: contrib/terminator/template.py (hunk 4 above)
Comment (yopito) [reply contrib/terminator/template.py]: I've tried, see additional checkdepends. But the testsuite crash, not all tests are at least runned. So I keep check disabled.

File: contrib/terminator/template.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    "pango",
+]
+pkgdesc = "Tiling terminal emulator application"
+maintainer = "yopito <pierre.bourgin@free.fr>"
+license = "GPL-2.0-only"
+url = "https://gnome-terminator.org"
+source = f"https://github.com/gnome-terminator/terminator/releases/download/v{pkgver}/terminator-{pkgver}.tar.gz"
+sha256 = "0ae9943f3d6b72230c14bcb355de84dd81274c033e76ca4698e80d7c93cd6ae5"
+hardening = ["vis"]
+# crossbuild: wait for crossbuilable deps
```
Comment (q66) [inline contrib/terminator/template.py:43]: deps not being crossbuildable is not grounds for disabling it

Comment (yopito) [reply contrib/terminator/template.py]: ok, fixed

**Pull request merged**